service/host.go

```diff
@@ -95,7 +95,7 @@ func (s *host) IsRunning() bool {
 func (s *host) OnCriticalError(err error) {
 	shutdown := true
 	if s.observer == nil {
-		s.Logger().Warn(
+		s.log.Warn(
```

Comment:
I was initially thinking to use s.log internally, only for setup purpose, but you are right that we should be consistent. I'll remove log object on the host itself.